### PR TITLE
Feedback on course

### PR DESCRIPTION
**Pull request by rupl**
@klamping requested feedback and I threw in a few copy edits! :tada:

Also, here are some additional notes that I didn't want to commit without discussion:
### Day 2
- npm install doesn't mention package.json or include `-g` flag (and I think using `-g` is bad form in general) so the modules might get installed in higher dirs. maybe package.json + `--save` flag?
### Day 4
- When opening the dropdown, reusing the `mainNav.selector` config would be a bit more DRY. Same with `loginForm` config. On the other hand, using var and then concatenating a string (e.g. `mainNav.selector + ' .about'`) is ugly.
### Day 5
- If people are looking for more examples, [WebdriverCSS has a Mocha assertion example](https://github.com/webdriverio/webdrivercss/blob/master/examples/webdrivercss.browserstack.with.mocha.js) written by yours truly :bowtie:
### Day 6
- The `wheels spinning` => `gears turning` — change of word choice simply because I normally associate "spinning your wheels" with my car being stuck in the mud or snow.

### CODE REVIEW

**Comment by micahgodbolt** [comment]
:+1: @rupl

**Comment by klamping** [comment]
@rupl awesome job with the review/edits. This is super helpful.

**Day 2:** I'm not sure the best way to handle it. I'm hoping to avoid having to run `npm init` and explain the `package.json` file, and as you mentioned, using `-g` is bad form. From my testing, running just `npm install whatever` with no `package.json` file will just create a new `node_modules` folder in the same directory. Maybe it's worth adding a note about needing the extra stuff if you want to re-use the code across computers.

**Day 4:** Good catch on using `mainNav.selector` for the click actions. Makes sense to re-use that var. Maybe I'll create some extra vars for the other selectors, so that at least there's some good practices in using vars. Any more complicated and you start drifting in to Page Objects, which is a deep subject to dive in to. Here's what I'm thinking:

```
var usernameSelector = ".login .username";
browser
    .webdrivercss("Login Default", loginForm)
    .setValue(usernameSelector, "admin")
```

**Day 5:** Excellent, we should definitely link to that.

**Day 6:** Fair point. While sometimes visual regression testing does feel like spinning wheels, it's definitely not what I was going for.

File: day-1.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ There are many types of software testing out there; [Wikipedia has an extensive
 
 In this course, we're focused on "functional" testing. This type of testing checks the functionality of your website and ensures that it works properly. It does not mean you're testing JavaScript "functions" (that would be unit testing).
 
-Frankly, you could probably call it "system testing" or "end-to-end tests", but for our course, we'll be using the term "functional testing".
+Frankly, you could probably call it "system testing" or "end-to-end tests," but for our course we'll be using the term "functional testing."
```
Comment (klamping) [inline day-1.md:41]: I keep forgetting if the punctuation goes in or out of the quotes...

**Comment by rupl** [comment]
> From my testing, running just npm install whatever with no package.json file will just create a new node_modules folder in the same directory.

This is also my normal expectation, but when I ran the command I ended up with a blank dir, and looking around I found a rando `node_modules` folder containing WebdriverIO one dir higher in `~/www` (which admittedly had been there before I ran the command for your stuff)

I allegedly know what I'm doing and managed to screw it up ¯\_(ツ)_/¯

**Comment by klamping** [comment]
Just looked up the docs for `npm init` and it looks like we can skip through the prompts and create a default package.json file. I think that's good enough for our needs:

run `npm init -y`
run `npm install --save ...`

I'll add some extra explanation to summarize what's going on. 

In the meantime, going to merge this PR so we have all of its goodness added to the repo. Will get the other edits in tonight hopefully.

**Pull request merged**